Sources/active-win/main.swift

```diff
@@ -57,8 +85,10 @@ for window in windows {
 
 	// This can't fail as we're only dealing with apps
 	let app = NSRunningApplication(processIdentifier: appPid)!
+	let appName = window[kCGWindowOwnerName as String] as! String
+	let browserURL = getBrowserURL(appName)
```

Comment:
Executing AppleScript is slow, so we should only do it if the window is a browser window we know we can handle.

Reply:
Rearranged the order here. Now `let browserURLAppleScriptCommand = getActiveBrowserTabURLAppleScriptCommand(appName)` is called in place of `getBrowserURL`. If the result `browserURLAppleScriptCommand` is not `nil`, only then do I call `getActiveBrowserTabURL` to execute the AppleScript.